### DISCUSSION TITLE
Is there any example about cross-gradient joint inversion

### DISCUSSION THREAD

**Discussion by xichaoqiang** [gimli-org/pyGIMLi#261] [Show and tell]
### Problem description

Is there any example or tutorials about cross-gradient joint inversion (refract & ERT )?

**Comment by halbmy** [comment]
Very good question! Unfortunately, we don't have any example on cross-gradient joint inversion as performed by Jordi et al. (2020) but I put it on the list. There is also no example on structurally coupled cooperative inversion yet.

**Comment by xichaoqiang** [comment]
Thank you very much.

**Comment by halbmy** [comment]
Just as an update: I have the code but it was based on an older pyGIMLi version so it needs adaption. Still working on it, stay tuned.

You might have a look at a recent discussion paper by Giulia de Pascale et al. (2020):
https://tc.copernicus.org/preprints/tc-2020-306/
who are using the Jordi et al. (2020) code.

**Comment by xichaoqiang** [comment]
Thanks for your reply.This study is great. I am looking forward to these codes.

**Comment by halbmy** [comment]
Right now the codes are so complicating that we would spend time more time discussing the problems step by step than creating a robust framework for it. But we're working on it. We just don't have a project or PhD student working with it.
